cypress/integration/tests/applicationinventory/application/create.test.ts

```diff
@@ -27,10 +29,18 @@ import { ApplicationInventory } from "../../../models/applicationinventory/appli
 import * as commonView from "../../../../integration/views/common.view";
 import * as data from "../../../../utils/data_utils";
 
-describe("Application validations", () => {
-    beforeEach("Login", function () {
+describe("Application validations", { tags: "@tier2" }, () => {
```

Comment:
This will be tier 1 right ? Crud , create and interlinked

Reply:
create is in tier 2